### PR TITLE
[#160725918] Office Blocks should be tied to a location

### PR DESCRIPTION
**Pull request by JoanNgatia**
### What does this PR do?
• Adds location field to office blocks and sets Kenya as the default location.

### Description of the task to be completed:
• Add location field on OfficeBlock model.
• Create migration to set the location of existent blocks to Kenya.

### How can this be manually tested?
Navigate to office blocks endpoint 

### Relevant PT stories:
[#160725692](https://www.pivotaltracker.com/story/show/160725918)

### CODE REVIEW

**Comment by JoanNgatia** [comment]
PR merge pending [this](https://github.com/AndelaOSP/art-backend/pull/190) PR's merge

File: core/migrations/0030_auto_20181015_1052.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.1.2 on 2018-10-15 10:52
```
Comment (esirK) [inline core/migrations/0030_auto_20181015_1052.py]: IMHO, I would suggest we have this migration and `0031_auto_20181015_1424.py` as one migration

File: core/migrations/populate_officeblock_location.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+
+def update_officeblocks_location(apps, schema_editor):
+    office_block = apps.get_model('core', 'OfficeBlock')
+    andela_centres = apps.get_model('core', 'AndelaCentre')
+    block_location = None
+    for office_block in office_block.objects.all():
+        # find kenya centre
+        centre = andela_centres.objects.filter(country="Kenya")[0]
```
Comment (esirK) [inline core/migrations/populate_officeblock_location.py]: I fear `andela_centres.objects.filter(country="Kenya")[0]` might fail with an **IndexError** if no center has a **country** named **kenya**. We could use `andela_centres.objects.filter(country="Kenya").exists()` instead.

**Comment by JoanNgatia** [comment]
@esirK Please approve changes suggested

**Pull request merged**